packages/x-charts/src/locales/ptBR.ts

```diff
@@ -0,0 +1,12 @@
+import { ChartsLocaleText } from './utils/chartsLocaleTextApi';
+import { getChartsLocalization } from './utils/getChartsLocalization';
+
+export const ptBRLocaleText: Partial<ChartsLocaleText> = {
```

Comment:
@JCQuintas please double-check this version, not sure what words are more common